lib/beaker/host/unix/pkg.rb

```diff
@@ -17,28 +17,30 @@ def check_for_command(name)
     end
   end
 
-  def check_for_package(name)
+  def check_for_package(name, opts = {})
+    opts = {:accept_all_exit_codes => true}.merge(opts)
+    opts[:prepend_cmds] ? pc = "#{opts[:prepend_cmds]} " : pc = ""
```

Comment:
Thanks for the suggestion.  Will rework the diffs and upload soon after testing.